packages/firebase_admob/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -8,11 +8,11 @@
 
```

Comment:
We don't typically include this file, but I need it to add `UnifiedNativeAdView.xib` to the xcode project. Otherwise, the app crashes when creating a `NativeAd`.